development/docker-compose.mysql.yml

```diff
@@ -15,7 +15,6 @@ services:
       - "NAUTOBOT_DB_ENGINE=django.db.backends.mysql"
   db:
     image: mysql:8
-    command: "--character-set-server=utf8mb4 --collation-server=utf8mb4_bin"
```

Comment:
Is this removed because mysql 8 already defaults to the correct server-side character set and collation?

Reply:
Yep!